src/knowledgebase/guidebook.yml

```
@@ -1129,3 +1129,23 @@ lost_passport:
         На время ожидания Вам должны дать Fiktionsbescheinigung, с которым сохраняется ваш 24й параграф и все связанные с ним права. В качестве удостоверения личности Вам выдадут Ausweisersatz - такая же складная зелёная бумажечка, как и Fiktionsbescheinigung, но с фотографией, поэтому советуем на термин взять фото.
         Позже, ближе к получению паспорта, нужно будет просить у них термин на вклейку  ВНЖ (с тем же сроком) в новый паспорт.
         #внж #lea #документ
+forms:
+  description: Каналы с переводами типовых форм и заявлений
+  contents:
+    - |
+      Каналы с переводами типовых форм и заявлений
+
+      Социал
+      https://t.me/Soz_AsylbLG_Grundsicherung
+
+      Джобцентр
+      https://t.me/Jobcenter_Soz_Formular_ru
```

```suggestion
      Sozialhilfe
      https://t.me/Soz_AsylbLG_Grundsicherung

      Jobcenter
      https://t.me/Jobcenter_Soz_Formular_ru
```
А то половина по-русски, половина по-немецки, неконсистентно))